benchmarks/config.properties

```diff
@@ -1,5 +1,5 @@
-inference_address=http://0.0.0.0:8080
-management_address=http://0.0.0.0:8081
+inference_address=http://127.0.0.1:8080
```

Comment:
Benchmarking also supports running Torchserve in a [container environment](https://github.com/pytorch/serve/blob/c69defd4bd79ddc73a475deecd0e0e2c2c37b6be/benchmarks/benchmark-ab.py#L365-L373). This config will not work in this case. I believe this doesn't break any of our current workflows though.